aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -324,25 +380,41 @@ impl<
         }
     }
 
-    pub fn set_base_values(&self, key: K, base_values: impl IntoIterator<Item = (T, V)>) {
+    pub fn set_raw_base_values(&self, key: K, base_values: impl IntoIterator<Item = (T, V)>) {
+        // Incarnation is irrelevant for storage version, set to 0.
+        self.group_values
+            .entry(key)
+            .or_default()
+            .set_raw_base_values(base_values.into_iter());
+    }
+
+    pub fn update_tagged_base_value_with_layout(
+        &self,
+        key: K,
+        tag: T,
+        value: V,
+        layout: Option<Arc<MoveTypeLayout>>,
```

Comment:
so it's none if V is a deletion? If so, can we assert here (since this is the pub interface for mvhashmap)?